src/ClassDesignChallenge/UniversityInfo.java

```diff
@@ -0,0 +1,26 @@
+package ClassDesignChallenge;
+
+import java.time.LocalDate;
+
+public class UniversityInfo {
+    public static void main(String[] args) {
+
+        Professors professor1 = new Professors("John", "Connor", 'M', LocalDate.of(1989, 12, 12), 33, 1891212050432L, "Canada, 8th Avenue, Apartment 12");
```

Comment:
If we want to make a search for a particular address, it will be easy do to it, having the entire address on a single string property?

Reply:
No, this is wrong and will complicate things. Will have a look in a moment, it seems it was rushed.

Reply:
New parameters have been created for adDress, more exactly, addressCity, addressStreet and addressApartmentNr. This might be easier when you need to search for an address by having more search options.